.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,6 +19,7 @@ What things should reviewers look out for?
 * [ ] Removed any instances of chunk naming.
 * [ ] Spell checked any Rmd file or md file.
 * [ ] Comments and documentation are up to date.
+* [ ] All links, particularly the "download this Rmd file" link have been tested.
```

Comment:
By tested here, do you mean by uploading the updated/new file in the PR into one's personal repo and testing the link that way? I ask because if the file is not already committed to master, one would get a the `Page not found` error.

Reply:
Ah. That's a fair point. hmm...

Reply:
I do like the idea of having a reminder to "check all of the links and ensure that they are properly formatted" (not in exact words) added to the checklist.

However, I cannot think of a particularly more efficient way to test the links before merging into master.

Reply:
I'm going to use your wording. And as far as the testing the links, maybe its a thing that can be automated too? Related to #228

Reply:
Basically this PR is a band-aid solution until we can do something for #228